Polygon_mesh_processing/test/Polygon_mesh_processing/triangulate_hole_Polyhedron_3_test.cpp

```diff
@@ -79,7 +79,7 @@ CGAL::internal::Weight_min_max_dihedral_and_area
     Halfedge_handle edge_it = halfedge(*begin, poly);
     double ang_max = 0;
     for(int i = 0; i < 3; ++i) {
-      double angle = 180 - CGAL::abs( 
+      double angle = CGAL_PI - CGAL::abs( 
```

Comment:
no further changes need?

Reply:
No, because this later checks if two weights computed with different methods are equal up to an epsilon. Both methods changed from degree to radian.